tool/update_blns.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:convert';
+import 'dart:io';
+
+String _blnsJsonRawUrl =
```

Comment:
final both of these? Don't need the type...

Reply:
Done.